packages/odyssey-storybook/src/components/odyssey-labs/SideNav/SideNav.stories.tsx

```diff
@@ -134,7 +134,7 @@ const storybookMeta: Meta<SideNavProps> = {
     },
   },
   args: {
-    appName: "Admin",
+    appName: "",
```

Comment:
Would we expect an empty string here? Or, would the consumer just not provide the prop?

Reply:
This is just a test, but I guess it can be either way. I will update the prop to be optional.